scripts/addBenefit.py

```diff
@@ -67,6 +73,35 @@ def moveExcelToStrapi():
   for item in data:
     call.postCollectionItem(strapiCollectionType, item)
 
+#updates strapi from csv
+##strapiKey - must match csvKey by value and be a field in Strapi
+##csvKey - must match strapiKey by value and be a header in the csv file
+##fields - must be a field in strapi and be in the header of the csv file
+def updateCSVRowsToStrapi(strapiKey, csvKey, fields):
+  call = CallStrapiAPI(user, password, strapi)
+
+  ##Ensure user is authenticated
+  if (not call.authenticate()):
+    print("Unable to Authenticate")
+    return
+
+  strapiItems = json.loads(call.getAllCollectionItems(strapiCollectionType))
+
+  csvParser = CSVParser(csvFile)
+  csvItems = csvParser.getData()
+
+  updatedItems = []
+  for strapiItem in strapiItems:
+    for csvItem in csvItems:
+      if strapiItem[strapiKey] == csvItem[csvKey]:
+        record = {"id": strapiItem["id"]}
+        for field in fields:
+          record[field] = csvItem[field]
+        updatedItems.append(record)
```

Comment:
I'm not going to lie, this hurts to look at. We need to add some unique identifier that would make this easier to map things from the CSV into Strapi.

Reply:
For the front end strapi needs a unique key amongst all verticals for benefit at a minimum. That should help clean the code up a bit for this script as well